wolfssl/openssl/x509.h

```diff
@@ -2,9 +2,6 @@
 
 #include <wolfssl/openssl/ssl.h>
 
-/* for compatibility, this is expected to be included */
-#include <openssl/dh.h>
```

Comment:
This is supposed to be including `wolfssl/openssl/dh.h`. Any reason we can't change it to that? I agree this is wrong, but its meant to be a wolf include.

Reply:
The comment about compatibility + the incomplete path made me think it could have been an external dependency.